File: MyBreak/DayThree/DayThree/JavaScript/Chapter4TaskFour.js
const prompt = require('prompt-sync')()
function salesCommissionCalculator() {
let numberOfItems = prompt("Enter the number amount you sold for last month  ")
let total = 0
total += numberOfItems

console.log("Index   \tValue")
for(let index = 0; index < 5; index++) {
  console.log(index + "  \t" + total)
}
}
salesCommissionCalculator()

File: MyBreak/DayThree/DayFour/Python/chapter4_task_four.py
def sales_commission_calculator():
  numbersOfItems = int(input("Enter the number of items you sold for last month "))
  total = 0
  total += numbersOfItems
  print("index  \tValue")
  for index in range(5):
    print(index + "\t" + total)
  
sales_commission_calculator()